cookbook/security/voters.rst

```diff
@@ -147,10 +147,10 @@ would look like this::
             }
 
             // you know $subject is a Post object, thanks to supports
-            /** @var Post $post */
+            /* @var Post $post */
```

Comment:
Change made by phpdoc_to_comment fixer (Symfony level)
Docblocks should only be used on structural elements.